nth-prime/nth_prime_test.rb

```diff
@@ -1,4 +1,22 @@
 require 'minitest/autorun'
+
+require 'prime'
+
+Error_message = "Using Ruby's Prime class is probably the best way to do this in a 'real'\n" + 
```

Comment:
I think the constant should either be CamelCase or SHOUT_CASE, but not a mix of Camel and snake_case.